demo-circuit/src/blaze_csw/constraints/data_structures.rs

```diff
@@ -1070,7 +1069,7 @@ impl CswFtProverDataGadget {
         );
 
         // Alloc phantom field element
-        let phantom_g = FieldElementGadget::from_value(cs.ns(|| "Break"), &PHANTOM_FIELD_ELEMENT);
+        let phantom_g = FieldElementGadget::from_value(cs.ns(|| "Break"), &FieldElement::default());
```

Comment:
Where is this used ?

Reply:
@DanieleDiBenedetto if you mean the phantom gadget, it is used to handle lists of commitment hashes whose size is less then the maximum size:

```
// Ignore NULL hashes
let should_ignore_hash = self.sc_txs_com_hashes_g[i].is_eq(
    cs.ns(|| format!("sc_txs_com_hashes[{}] == PHANTOM", i)),
    &phantom_g,
)?;
```